app/src/main/java/org/mozilla/fenix/home/mozonline/PrivacyContentDisplayHelper.kt

```diff
@@ -44,15 +47,19 @@ fun showPrivacyPopWindow(context: Context, activity: Activity) {
     // Users can only use fenix after they agree with the privacy notice
     val builder = AlertDialog.Builder(activity)
         .setPositiveButton(
-            context.getString(R.string.privacy_notice_positive_button),
-            { _, _ ->
-                context.settings().shouldShowPrivacyPopWindow = false
+            context.getString(R.string.privacy_notice_positive_button)
+        ) { _, _ ->
+            context.settings().shouldShowPrivacyPopWindow = false
+            context.settings().preferences.edit {
+                putBoolean(context.getString(R.string.pref_key_marketing_telemetry), true)
             }
-        )
+            if (context.settings().isMarketingTelemetryEnabled) {
```

Comment:
If we set `isMarketingTelemetryEnabled = true` above (or even by editing the preference manually) then this `if` check is not needed.